### ISSUE TITLE
Explanation of using the tool

### ISSUE THREAD

**Issue by ghost** [MajorTanya/mihon-dummyextension#1]
I wonder if there is an explanation for actually using the accessory. Until now, I do not know how to use it. If possible, a video explanation of the accessory

**Comment by MajorTanya** [comment]
This is just meant as a debugging help for stuff that affects extension install/update/uninstall in Mihon and its related forks. There is no "use" for it other than that. If you check the code, it doesn't support doing anything with it. It will show a "This is a non-functioning dummy extension" error if you try to do anything, like searching or browsing with it.

The extension is updated every so often (currently about once a week, probably more often in future) to simulate an extension update from an extension repo. This can help during debugging in, say, Mihon, because you can't be sure that third-party extensions from other repositories will ever be updated in time for your debugging work.

TL;DR: If you're not working on debugging extension installs/updates/removals for Mihon or a related fork, this repo is useless for you. 👍
